content/en/blog/_posts/data-tbd/index.md

```diff
@@ -0,0 +1,281 @@
+---
+layout: blog
+title: "Dynamic Resource Allocation in Kubernetes 1.26"
+date: 
+slug: dynamic-resource-allocation
+---
+
+ **Authors:** Patrick Ohly (Intel), Kevin Klues (NVIDIA)
+
+Dynamic resource allocation is a new API for requesting and sharing resources
+between pods and containers inside a pod. It is a generalization of the
+persistent volumes API for generic resources. Third-party resource drivers are
+responsible for tracking and allocating resources. Different kinds of
+resources support arbitrary parameters for defining requirements and
+initialization.
+
+Dynamic resource allocation is an *alpha feature* and only enabled when the
+`DynamicResourceAllocation` [feature
+gate](/docs/reference/command-line-tools-reference/feature-gates/) and the
+`resource.k8s.io/v1alpha1` {{< glossary_tooltip text="API group"
+term_id="api-group" >}} are enabled. For details on that, see the
+`--feature-gates` and `--runtime-config` [kube-apiserver
+parameters](/docs/reference/command-line-tools-reference/kube-apiserver/).
+kube-scheduler, kube-controller-manager and kubelet also need the feature gate.
```

Comment:
"See the `--feature-gates` and `--runtime-config` kube-apiserver parameters." seems like a normal sentence to me.

I can leave out the second sentence, but I know from experience that one of the mistakes that users make is not setting the feature gate for all components. In particular kubelet is a candidate for this.

Reply:
I see it now, I didn't parse that `kube-apiserver parameters` was part of the sentence because of the way it was wrapping. That sounds fine.

For the second sentence, i agree it is important now that I understand what is trying to be said. Let's just change it slightly to:
> The kube-scheduler, kube-controller-manager and kubelet components all need the feature gate enabled as well.

Reply:
Changed.